kettle-plugins/kafka/src/main/java/org/pentaho/big/data/kettle/plugins/kafka/KafkaConsumerInput.java

```diff
@@ -101,4 +112,29 @@ private void commitOffsets( Map.Entry<List<List<Object>>, Result> rowsAndResult
     ( (KafkaStreamSource) source ).commitOffsets( rowsAndResult.getKey() );
   }
 
+  private boolean checkKafkaConnectionStatus( KafkaConsumerInputMeta meta ) {
+    boolean kafkaConnectionStatus = false;
+    Properties props = new Properties();
+    props.put( ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, meta.getBootstrapServers()  );
+    props.put( ConsumerConfig.GROUP_ID_CONFIG,  meta.getConsumerGroup()  );
+    props.put( ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class );
+    props.put( ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class );
+    meta.getConfig().entrySet()
+      .forEach( ( entry -> props.put( entry.getKey(),
+        (String) entry.getValue() ) )  );
+
+    AdminClient client = AdminClient.create(props);
+    Collection<Node> nodes = null;
+    try {
+      nodes = client.describeCluster().nodes().get();
+    }catch( ExecutionException e ) {
```

Comment:
Should be a space after the } before catch.  Please double-check spacing in other places too.